src/subscription/__tests__/eventEmitterAsyncIterator.js

```diff
@@ -0,0 +1,74 @@
+/**
+ *  Copyright (c) 2017, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
```

Comment:
This file is only used for tests. It's an example of a mapping between `EventEmitter` and `AsyncIterator`. Especially since graphql-subscriptions uses `EventEmitter` internally now, this file could help bridge the gap between the interfaces.